src/definitions/transform.ts

```diff
@@ -26,6 +27,7 @@ const transform: {[key in TransformName]: Transform} = {
   toLowerCase: (s) => s.toLowerCase(),
   toUpperCase: (s) => s.toUpperCase(),
   toEnumCase: (s, cfg) => cfg?.hash[configKey(s)] || s,
+  normalize: (s) => s.replace(/\s\s+/g, " "),
```

Comment:
why not just `\s+` ?

Reply:
...or `[\s\t]+` if tabs were to be included too

Reply:
because `\s+` means single space or more, and in my case i want to only check multiple spaces and reduce them to one space

Reply:
also, pretty much the regex `\s\s+` catters for all spaces, tabs and line breaks, maybe i add more tests for these other scenarios, but will also improve the regex to `/(\s\s+)|\t|\n/` for our own peace of mind

Reply:
The performance of replacing `/\s+/` can be better than `/\s\s+/` and the result will be exactly the same.

Reply:
check my latest commits, i have resolved these issues and also added `trimInner` option